src/components/mainContent/Navbar.css

```diff
@@ -0,0 +1,40 @@
+.dropdown {
+  position: relative;
+  display: inline-block;
+  width: 100%;
+  height: 450px;
+}
+
+.dropdown-content {
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  border-radius: 0 0 10px 10px;
+  display: none;
+  position: absolute;
+  background-color: #f7f7f7;
+  min-width: 160px;
+  box-shadow: 0px 8px 16px 0px 898F9C;
+  z-index: 1;
+  text-align: center;
+  top: 46px;
+  left: -63px;
+}
+
+.dropdown-content a {
+  color: black;
+  padding: 12px 16px;
+  text-decoration: none;
+  display: block;
+  font-family: 'Mulish';
+  font-size: 1.3rem;
+  font-weight: bold;
+  color: #4267B2;
+  text-align: center;
```

Comment:
Typically, dropdowns are right or left-aligned - this one is up to you though!

Reply:
Oh alright, the Figma design seemed to have centered text so I based the text alignment on that!